yt_dlp/extractor/beatbump.py

```diff
@@ -0,0 +1,112 @@
+import re
+
+from yt_dlp.utils import ExtractorError
+from .common import InfoExtractor
+
+
+class BeatBumpIE(InfoExtractor):
+    _VALID_URL = r'https:\/\/beatbump\.ml\/(?:(?:listen\?id=)|(?:playlist\/)|(?:artist\/)|(?:release\?id=))(?P<id>.*)'
+    _TESTS = [{
+        # Single video
+        'url': 'https://beatbump.ml/listen?id=BciS5krYL80',
+        'md5': '7f6c65ee0955228bcbb81066080fb46c',
+        'info_dict': {
+            'id': 'BciS5krYL80',
+            'ext': 'mp4',
+            'title': 'Hotel California (2013 Remaster)',
+            'thumbnail': 'https://i.ytimg.com/vi_webp/BciS5krYL80/maxresdefault.webp',
+            'artist': 'Eagles',
+            'track': 'Hotel California (2013 Remaster)',
+            'channel_follower_count': int,
+            'channel_url': 'https://www.youtube.com/channel/UC49r4GNHHpc-eQ9hmD2Rg6A',
+            'uploader': 'The Eagles - Topic',
+            'alt_title': 'Hotel California (2013 Remaster)',
+            'categories': ['Music'],
+            'like_count': int,
+            'view_count': int,
+            'uploader_id': 'UC49r4GNHHpc-eQ9hmD2Rg6A',
+            'tags': ['Eagles', 'Legacy', 'Hotel California'],
+            'upload_date': '20181101',
+            'release_year': 1976,
+            'duration': 391,
+            'channel_id': 'UC49r4GNHHpc-eQ9hmD2Rg6A',
+            'creator': 'Eagles',
+            'description': 'md5:1846212e0e373e0970f844516a9ced39',
+            'playable_in_embed': True,
+            'uploader_url': 'http://www.youtube.com/channel/UC49r4GNHHpc-eQ9hmD2Rg6A',
+            'album': 'Legacy',
+            'age_limit': 0,
+            'channel': 'Eagles',
+            'availability': 'public',
+        }
+    }, {
```

Comment:
1. Add `'add_ie': ['Youtube'],` for this test and `'add_ie': ['YoutubeTab'],` for others
2. Remove the whole `info_dict` for all tests; they're not from this IE

Reply:
> Remove the whole info_dict for all tests; they're not from this IE

I think this is fine - it needs to test that the url is sent to the youtube extractor correctly and that it understands it. Tests that resolve to other extractors are fairly common.